### PR TITLE
changes to remove rancher_manager_support setting

### PR DESCRIPTION
**Pull request by ibrokethecloud**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**
PR introduces changes to drop setting for `rancher-manager-support` setting.

This is not needed as it has been dropped in favor of the vcluster addon.

**Problem:**

**Solution:**

**Related Issue:**

**Test plan:**

### CODE REVIEW

File: pkg/webhook/resources/setting/validator.go [hunk 1]
```diff
@@ -14,9 +14,6 @@ import (
 	"strings"
 
 	"github.com/longhorn/backupstore"
-	// Although we don't use following drivers directly, we need to import them to register drivers.
```
Comment (guangbochen) [inline pkg/webhook/resources/setting/validator.go]: please keep this, otherwise the backup validation would fail

Comment (ibrokethecloud) [reply pkg/webhook/resources/setting/validator.go]: 👍

**Review by guangbochen — APPROVED** [review]
LGTM, verified that the Harvester API server is running correctly and contains no outstanding error logs.

**Review by chrisho — APPROVED** [review]
LGTM, thanks.

**Comment by bk201** [comment]
Should we also revert https://github.com/harvester/harvester/pull/3779? It's Ok to do it in another PR.

**Review by bk201 — APPROVED** [review]

**Comment by guangbochen** [comment]
@mergify backport v1.2

**Pull request merged**